src/java/boa/types/proto/VariableProtoTuple.java

```diff
@@ -1,6 +1,7 @@
 /*
```

Comment:
This file actually shouldnt be changed, because there was no 'specs' attribute added to it.  Please revert all changes to this class.